application/cypress/integration/card/card.spec.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jest/expect-expect */
+/* eslint-disable jest/valid-expect */
+/// <reference types="cypress" />
```

Comment:
You can adjust your jest/test config/settings instead of placing that in every test

Reply:
fixed